db/migrate/20190718233448_add_foreign_key_constraints.rb

```diff
@@ -0,0 +1,23 @@
+class AddForeignKeyConstraints < ActiveRecord::Migration[5.1]
+  def change
+    add_foreign_key "amr_counts", "pipeline_runs", name: "amr_counts_pipeline_run_id_fk", on_delete: :cascade
+    add_foreign_key "ercc_counts", "pipeline_runs", name: "ercc_counts_pipeline_run_id_fk", on_delete: :cascade
```

Comment:
I think we shouldn't enable this one yet (ercc_counts has 2.5M records). I would add foreign keys in batches (one per week) and monitor to see if we have any big performance issue.